packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -2106,6 +2190,25 @@ def sum(
         #     max_vals=max_val,
         # )
 
+    def mean(
+        self,
+        axis: Optional[Union[int, Tuple[int, ...]]] = None,
+        **kwargs: Any,
+    ) -> PhiTensor:
+        child = self.child.mean(axis=axis, **kwargs)
+        data_subjects = np.array(self.data_subjects.mean(axis=axis, **kwargs))
```

Comment:
MAYBE: We could do 
`data_subjects=self.data_subjects.mean(axis, **kwargs)`
as data_subjects we visualize as a numpy array with basic operation needed to perform mean.